other_checks/redis.sp

```diff
@@ -0,0 +1,17 @@
+locals {
+  other_redis_common_tags = merge(local.azure_compliance_common_tags, {
+    service = "Azure/Redis"
+  })
+}
+
+benchmark "other_redis" {
+  title = "Redis"
+  children = [
+    control.azure_redis_cache_in_virtual_network,
```

Comment:
```suggestion
    control.redis_cache_in_virtual_network,
```
@khushboo9024 Should we remove the work azure from this to be in naming consistency?